cfme/automate/import_export.py

```diff
@@ -71,6 +71,7 @@ def is_displayed(self):
 
 
 class FileImportSelectorView(AutomateImportExportBaseView):
+    flash = V2VFlashMessages(locator="div.import-flash-message")
```

Comment:
I'm curious, why are you using`V2VFlashMessages`?

Reply:
```FlashMessages``` widget does not work for error messages on this page. Hence used this widget.

Reply:
@ganeshhubale, what exactly didn't work ?

Reply:
- There is issue with xpath. it doesn't work with  ```locator="div.import-flash-message"```. As it gives none while reading flash message. 
- And it gets  covered in this widget with xpath: 
```msg_xpath = './/div[contains(@class, "alert")]'```

Hence used ```V2VFlashMessages```

Reply:
@ganeshhubale, @mshriver, I guess it's worth merging these two widgets then. either way, I don't expect to see this change in current PR.